src/forms/BoundDateRangeField.tsx

```diff
@@ -32,8 +32,11 @@ export function BoundDateRangeField(props: BoundDateRangeFieldProps) {
           label={label}
           value={field.value || undefined}
           onChange={(value) => {
-            onChange(value);
-            field.maybeAutoSave();
+            // DateRangeField will return an undefined if part of filter, but we're not doing that here yet
```

Comment:
**question:** Any reason not to do that here yet?

Reply:
None anymore, if we're going to allow `DateRangeField` to support undefined in general.